guardrails/cli/configure.py

```diff
@@ -14,23 +14,27 @@
 
 DEFAULT_TOKEN = ""
 DEFAULT_ENABLE_METRICS = True
+DEFAULT_USE_REMOTE_INFERENCING = True
 
 
 def save_configuration_file(
-    token: Optional[str], enable_metrics: Optional[bool]
+    token: Optional[str], enable_metrics: Optional[bool], use_remote_inferencing: bool
```

Comment:
If use_remove_inferencing can be None, as evidenced by line 27, this should be Optional[bool].

Reply:
good catches on stuff like this, thanks